experimental/dds/attributable-map/package.json

```diff
@@ -100,6 +142,18 @@
 		"rimraf": "^4.4.0",
 		"typescript": "~5.1.6"
 	},
+	"fluidBuild": {
+		"tasks": {
+			"build:docs": {
+				"dependsOn": [
+					"...",
+					"api-extractor:commonjs",
+					"api-extractor:esnext"
+				],
+				"script": false
+			}
+		}
+	},
```

Comment:
We should be able to get rid of these overrides in the per-package fluid-build config now. There was some blocker but I think was resolved. I'll follow up separately.

Reply:
I'll try excluding them in my next PR.  (Cleanup of existing packages now tracked by AB#7418.)